### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by zhouqiang-cl**

### CODE REVIEW

**Comment by zhouqiang-cl** [comment]
@mahjonp PTAL

File: README.md [hunk 1]
```diff
@@ -63,19 +79,28 @@ For example:
 ./bin/go-tpc tpcc --warehouses 4 prepare --output-type csv --output-dir data --pprof :10111
 ```
 
-If you want to import tpcc data into TiDB, please refer to [import-to-tidb](docs/import-to-tidb.md).
+If you want to import tpcc csv data into TiDB, please refer to [import-to-tidb](docs/import-to-tidb.md).
```
Comment (yeya24) [inline README.md:82]: This change is not needed since in that document I covered both csv and non-csv ways.

File: README.md [hunk 2]
```diff
@@ -43,18 +43,34 @@ For example:
 
 ### TPC-C
 
-
+#### prepare
```
Comment (mahjonp) [inline README.md:46]:
```suggestion
#### Prepare
```

File: README.md [hunk 3]
```diff
@@ -43,18 +43,34 @@ For example:
...
 ```bash
 # Create 4 warehouses and use 4 partitions by HASH 
 ./bin/go-tpc tpcc --warehouses 4 --parts 4 prepare
-# Run TPCC workloads
+```
+
+#### run
```
Comment (mahjonp) [inline README.md:52]: ditto.

File: README.md [hunk 4]
```diff
@@ -43,18 +43,34 @@ For example:
...
+```bash
```
Comment (mahjonp) [inline README.md:53]: Need a blank line here.

**Pull request merged**